### ISSUE TITLE
Uncaught TypeError: Path must be a string. Received undefined

### ISSUE THREAD

**Issue by bcbcarl** [AtomLinter/linter-stylelint#358]
Steps to reproduce:

1. Open Atom.
2. Throws a TypeError exception.

**Atom**: 1.18.0 x64
**Electron**: 1.3.15
**OS**: Mac OS X 10.12.4
**Thrown From**: [linter-stylelint](https://github.com/AtomLinter/linter-stylelint) package 4.0.0

### Stack Trace

Uncaught TypeError: Path must be a string. Received undefined

```
At path.js:7

TypeError: Path must be a string. Received undefined
    at assertPath (path.js:7:11)
    at Object.resolve (path.js:1148:7)
    at eachParent (/packages/linter-stylelint/node_modules/browserslist/index.js:65:18)
    at getStat (/packages/linter-stylelint/node_modules/browserslist/index.js:79:12)
    at browserslist (/packages/linter-stylelint/node_modules/browserslist/index.js:188:15)
    at Browsers.parse (/packages/linter-stylelint/node_modules/autoprefixer/lib/browsers.js:61:16)
    at /packages/linter-stylelint/node_modules/autoprefixer/lib/browsers.js:52:30)
    at /packages/linter-stylelint/node_modules/stylelint/lib/utils/isAutoprefixable.js:19:3)
    at Module.get_Module._compile (/Applications/Atom.app/Contents/Resources/app/src/native-compile-cache.js:106:36)
    at Object.value [as .js] (/Applications/Atom.app/Contents/Resources/app/src/compile-cache.js:239:29)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (Applications/Atom.app/Contents/Resources/app/static/index.js:47:45)
    at require (/Applications/Atom.app/Contents/Resources/app/src/native-compile-cache.js:66:33)
    at /packages/linter-stylelint/node_modules/stylelint/lib/rules/at-rule-no-vendor-prefix/index.js:3:26)
    at Module.get_Module._compile (/Applications/Atom.app/Contents/Resources/app/src/native-compile-cache.js:106:36)
    at Object.value [as .js] (/Applications/Atom.app/Contents/Resources/app/src/compile-cache.js:239:29)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (Applications/Atom.app/Contents/Resources/app/static/index.js:47:45)
    at require (/Applications/Atom.app/Contents/Resources/app/src/native-compile-cache.js:66:33)
    at /packages/linter-stylelint/node_modules/stylelint/lib/rules/index.js:9:30)
    at Module.get_Module._compile (/Applications/Atom.app/Contents/Resources/app/src/native-compile-cache.js:106:36)
    at Object.value [as .js] (/Applications/Atom.app/Contents/Resources/app/src/compile-cache.js:239:29)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (Applications/Atom.app/Contents/Resources/app/static/index.js:47:45)
```

### Non-Core Packages

```
atom-ternjs 0.18.3 
busy-signal 1.4.3 
editorconfig 2.2.2 
file-icons 2.1.9 
git-time-machine 1.5.9 
intentions 1.1.2 
language-docker 1.1.8 
language-markdown 0.25.1 
linter 2.2.0 
linter-clang 4.1.1 
linter-eslint 8.2.1 
linter-stylelint 4.0.0 
linter-ui-default 1.6.3 
merge-conflicts 1.4.5 
react 0.16.2 
sort-lines 0.14.0 
vim-mode-plus 0.94.0 
```

**Comment by firefoxic** [comment]
The same error on:
**Atom**: 1.18.0 x64
**Electron**: 1.4.16
**OS**: ArchLinux (kernel 4.11.9)
**Thrown From**: linter-stylelint package 4.0.0

**Comment by iwinux** [comment]
Same here. Atom 1.18.0 with linter-stylelint 4.0.0

**Comment by retrixe** [comment]
Confirmed.

**Comment by viviancpy** [comment]
Same here. 
Atom 1.18.0 x64
linter-stylelint 4.0.0

**Comment by ntwb** [comment]
I'm pretty sure this is due to https://github.com/ai/browserslist/pull/161

Can you update your dependencies so that browserlist gets updated to 2.2.2 and test?

**Comment by firefoxic** [comment]
How can I do this without breaking anything?

**Comment by iwinux** [comment]
```sh
$ cd ~/.atom/packages/linter-stylelint
$ npm install browserslist@2.2.2
```

Seems to fix it.

**Comment by retrixe** [comment]
@iwinux yeah, I figured that out, just forked and will submit a PR :stuck_out_tongue_closed_eyes:

**Comment by Arcanemagus** [comment]
Fix published in v4.0.1, thanks again @retrixe!

If for some reason this doesn't work for you, try re-installing the package completely to ensure that Atom is re-installing the dependencies.
